app/javascript/packages/document-capture/hooks/use-cookie.js

```diff
@@ -0,0 +1,59 @@
+import { useState, useEffect } from 'react';
+
+/**
+ * Returns the current document cookie as an object.
+ *
+ * @return {Record<string, string>}
+ */
+function getCookieObject() {
+  const { cookie } = document;
+  return cookie
+    .split(';')
+    .map((part) => part.trim().split('='))
+    .filter(([_key, value]) => typeof value === 'string')
+    .reduce((result, [key, value]) => Object.assign(result, { [key]: value }), {});
+}
```

Comment:
One thought, since we're always using this helper for a specific cookie value name, do we need to parse out the whole object? It's probably fast enough it doesn't matter

Reply:
Interesting thought. Would you imagine something like a regexp to pull the specific value we're interested in?

```js
cookie.match(new RegExp(`(?:^|; )${name}=([^;]|$)+`))?.[1]
```

Or maybe just filtering the parts to the one we care about:

```js
cookie.split(';').map((part) => part.trim().split('=')).find(([key]) => key === name)?.[1]
```

Reply:
I think I would have done something like the latter, to minimize errors due to regexes

Reply:
> I think I would have done something like the latter, to minimize errors due to regexes

👍  Updated in f00a2a5.